projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -1335,7 +1335,7 @@ private boolean getContainer(List<String> options, List<String> parameters) {
     String containerName = parameters.get(0);
     Container container = _workHelper.getContainer(containerName);
     if (container != null) {
-      _logger.output(container.getTestrigs() + "\n");
```

Comment:
For now, it should print something like "Container{name=foo, testrigsUri=xxx}" (tested in container's toString method). This is not a proper message here, will change it after decide the representation of a Container.